deno.json

```diff
@@ -1,5 +1,17 @@
 {
   "tasks": {
-    "ok": "deno lint && deno fmt --check"
+    "ok": "deno task fmt && deno task lint && deno task check",
+    "check": "deno check **/*.ts",
+    "fmt": "deno fmt --check && deno task fmt:licence-headers",
+    "fmt:licence-headers": "deno run --allow-read --allow-write ./_tools/check_licence.ts",
+    "lint": "deno lint && deno task lint:doc-imports",
+    "lint:doc-imports": "deno run --allow-env --allow-read ./_tools/check_doc_imports.ts"
+  },
+  "lock": false,
```

Comment:
Good question! It turns off lockfile generation which is useful for us from a library support point of a view as ideally we'll want to trust semver rather than pinning to a particular version! wdyt?

Reply:
I like it! Makes sense.